## Trending Bioinformatics Repositories (Updated: 2025-09-26)

**Showing 60 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** â€“ â­ 6226 â€“ Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[ArcInstitute/evo2](https://github.com/ArcInstitute/evo2)** â€“ â­ 3100 â€“ Genome modeling and design across all domains of life
- **[satijalab/seurat](https://github.com/satijalab/seurat)** â€“ â­ 2523 â€“ R toolkit for single cell genomics
- **[lh3/minimap2](https://github.com/lh3/minimap2)** â€“ â­ 2040 â€“ A versatile pairwise aligner for genomic and spliced nucleotide sequences
- **[rust-bio/rust-bio](https://github.com/rust-bio/rust-bio)** â€“ â­ 1741 â€“ This library provides implementations of many algorithms and data structures that are useful for bioinformatics. All provided implementations are rigorously tested via continuous integration.
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** â€“ â­ 1725 â€“ Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** â€“ â­ 1533 â€“ A library of free open source icons for science illustrations in biology and chemistry
- **[evo-design/evo](https://github.com/evo-design/evo)** â€“ â­ 1419 â€“ Biological foundation modeling from molecular to genome scale
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** â€“ â­ 1356 â€“ Aggregate results from bioinformatics analyses across many samples into a single report.
- **[vgteam/vg](https://github.com/vgteam/vg)** â€“ â­ 1240 â€“ tools for working with genome variation graphs
- **[thieu1995/mealpy](https://github.com/thieu1995/mealpy)** â€“ â­ 1089 â€“ A Collection Of The State-of-the-art Metaheuristic Algorithms In Python (Metaheuristic/Optimizer/Nature-inspired/Biology)
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** â€“ â­ 1062 â€“ Collection of bioinformatics training materials
- **[pachterlab/gget](https://github.com/pachterlab/gget)** â€“ â­ 1056 â€“ ğŸ§¬ gget enables efficient querying of genomic reference databases
- **[hail-is/hail](https://github.com/hail-is/hail)** â€“ â­ 1027 â€“ Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** â€“ â­ 1010 â€“ A list of interesting genome browser and genome visualization programs
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** â€“ â­ 973 â€“ scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[ablab/spades](https://github.com/ablab/spades)** â€“ â­ 869 â€“ SPAdes Genome Assembler
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** â€“ â­ 862 â€“ cBioPortal for Cancer Genomics
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** â€“ â­ 861 â€“ A comprehensive library for computational molecular biology
- **[pysam-developers/pysam](https://github.com/pysam-developers/pysam)** â€“ â­ 853 â€“ Pysam is a Python package for reading, manipulating, and writing genomics data such as SAM/BAM/CRAM and VCF/BCF files. It's a lightweight wrapper of the HTSlib API, the same one that powers samtools, bcftools, and tabix.
- **[BioContainers/containers](https://github.com/BioContainers/containers)** â€“ â­ 750 â€“ Bioinformatics containers
- **[igvteam/igv](https://github.com/igvteam/igv)** â€“ â­ 700 â€“ Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[marbl/canu](https://github.com/marbl/canu)** â€“ â­ 691 â€“ A single molecule sequence assembler for genomes large and small.
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** â€“ â­ 687 â€“ Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** â€“ â­ 616 â€“ Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** â€“ â­ 609 â€“ Official home of genome aligner based upon notion of Cactus graphs
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** â€“ â­ 559 â€“ Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[oschwengers/bakta](https://github.com/oschwengers/bakta)** â€“ â­ 552 â€“ Rapid & standardized annotation of bacterial genomes, MAGs & plasmids
- **[Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk)** â€“ â­ 543 â€“ GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** â€“ â­ 523 â€“ Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[lindenb/jvarkit](https://github.com/lindenb/jvarkit)** â€“ â­ 512 â€“ Java utilities for Bioinformatics
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** â€“ â­ 505 â€“ The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** â€“ â­ 494 â€“ Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[bactopia/bactopia](https://github.com/bactopia/bactopia)** â€“ â­ 475 â€“ A flexible pipeline for complete analysis of bacterial genomes
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** â€“ â­ 467 â€“ PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[cov-lineages/pangolin](https://github.com/cov-lineages/pangolin)** â€“ â­ 441 â€“ Software package for assigning SARS-CoV-2 genome sequences to global lineages.
- **[marbl/Mash](https://github.com/marbl/Mash)** â€“ â­ 419 â€“ Fast genome and metagenome distance estimation using MinHash
- **[evanpeikon/Bioinformatics_Toolkit](https://github.com/evanpeikon/Bioinformatics_Toolkit)** â€“ â­ 415 â€“ A growing collection of bioinformatics tutorials, project guides, tools, and articles Iâ€™ve developed to make bioinformatics more accessible.
- **[zeqianli/tgv](https://github.com/zeqianli/tgv)** â€“ â­ 405 â€“ Explore genomes in the terminal. Light, blazing fast ğŸš€, vim-motion.
- **[theislab/scarches](https://github.com/theislab/scarches)** â€“ â­ 384 â€“ Reference mapping for single-cell genomics
- **[lh3/miniprot](https://github.com/lh3/miniprot)** â€“ â­ 379 â€“ Align proteins to genomes with splicing and frameshift
- **[moshi4/pyGenomeViz](https://github.com/moshi4/pyGenomeViz)** â€“ â­ 363 â€“ A genome visualization python package for comparative genomics
- **[ncbi/pgap](https://github.com/ncbi/pgap)** â€“ â­ 355 â€“ NCBI Prokaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** â€“ â­ 346 â€“ Genome-scale protein function classification
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** â€“ â­ 345 â€“ Tools for working with genomic and high throughput sequencing data.
- **[NVIDIA-Genomics-Research/rapids-single-cell-examples](https://github.com/NVIDIA-Genomics-Research/rapids-single-cell-examples)** â€“ â­ 344 â€“ Examples of single-cell genomic analysis accelerated with RAPIDS
- **[rrwick/Trycycler](https://github.com/rrwick/Trycycler)** â€“ â­ 333 â€“ A tool for generating consensus long-read assemblies for bacterial genomes
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** â€“ â­ 319 â€“ :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** â€“ â­ 318 â€“ Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[xiaolei-lab/rMVP](https://github.com/xiaolei-lab/rMVP)** â€“ â­ 310 â€“ :postbox: A Memory-efficient, Visualization-enhanced, and Parallel-accelerated Tool For Genome-Wide Association Study
- **[GenerTeam/GENERanno](https://github.com/GenerTeam/GENERanno)** â€“ â­ 301 â€“ GENERanno: A Genomic Foundation Model for Metagenomic Annotation
- **[WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM)** â€“ â­ 295 â€“ Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** â€“ â­ 289 â€“ Genomic Pre-trained Network
- **[projectglow/glow](https://github.com/projectglow/glow)** â€“ â­ 287 â€“ An open-source toolkit for large-scale genomic analysis
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** â€“ â­ 280 â€“ Build a partitioned pangenome graph from microbial genomes
- **[KamilSJaron/smudgeplot](https://github.com/KamilSJaron/smudgeplot)** â€“ â­ 277 â€“ Inference of ploidy and heterozygosity structure using whole genome sequencing data
- **[tjiangHIT/cuteSV](https://github.com/tjiangHIT/cuteSV)** â€“ â­ 272 â€“ Long read based human genomic structural variation detection with cuteSV
- **[sigven/pcgr](https://github.com/sigven/pcgr)** â€“ â­ 270 â€“ Personal Cancer Genome Reporter (PCGR)
- **[GMOD/jbrowse-components](https://github.com/GMOD/jbrowse-components)** â€“ â­ 255 â€“ Source code for JBrowse 2, a modern React-based genome browser
- **[biowasm/biowasm](https://github.com/biowasm/biowasm)** â€“ â­ 252 â€“ WebAssembly modules for genomics
