## Trending Bioinformatics Repositories (Updated: 2026-01-16)

**Showing 55 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** â€“ â­ 6465 â€“ Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[satijalab/seurat](https://github.com/satijalab/seurat)** â€“ â­ 2612 â€“ R toolkit for single cell genomics
- **[hussius/deeplearning-biology](https://github.com/hussius/deeplearning-biology)** â€“ â­ 2114 â€“ A list of deep learning implementations in biology
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** â€“ â­ 1790 â€“ Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** â€“ â­ 1596 â€“ A library of free open source icons for science illustrations in biology and chemistry
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** â€“ â­ 1407 â€“ Aggregate results from bioinformatics analyses across many samples into a single report.
- **[vgteam/vg](https://github.com/vgteam/vg)** â€“ â­ 1290 â€“ tools for working with genome variation graphs
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** â€“ â­ 1135 â€“ scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[ai-boost/awesome-ai-for-science](https://github.com/ai-boost/awesome-ai-for-science)** â€“ â­ 1115 â€“ A curated list of awesome AI tools, libraries, papers, datasets, and frameworks that accelerate scientific discovery â€” from physics and chemistry to biology, materials, and beyond.
- **[pachterlab/gget](https://github.com/pachterlab/gget)** â€“ â­ 1094 â€“ ğŸ§¬ gget enables efficient querying of genomic reference databases
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** â€“ â­ 1089 â€“ Collection of bioinformatics training materials
- **[hail-is/hail](https://github.com/hail-is/hail)** â€“ â­ 1045 â€“ Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** â€“ â­ 1042 â€“ A list of interesting genome browser and genome visualization programs
- **[tseemann/prokka](https://github.com/tseemann/prokka)** â€“ â­ 949 â€“ :zap: :aquarius: Rapid prokaryotic genome annotation
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** â€“ â­ 927 â€“ cBioPortal for Cancer Genomics
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** â€“ â­ 916 â€“ A comprehensive library for computational molecular biology
- **[ablab/spades](https://github.com/ablab/spades)** â€“ â­ 897 â€“ SPAdes Genome Assembler
- **[tanghaibao/jcvi](https://github.com/tanghaibao/jcvi)** â€“ â­ 883 â€“ Python library to facilitate genome assembly, annotation, and comparative genomics
- **[pysam-developers/pysam](https://github.com/pysam-developers/pysam)** â€“ â­ 875 â€“ Pysam is a Python package for reading, manipulating, and writing genomics data such as SAM/BAM/CRAM and VCF/BCF files. It's a lightweight wrapper of the HTSlib API, the same one that powers samtools, bcftools, and tabix.
- **[instadeepai/nucleotide-transformer](https://github.com/instadeepai/nucleotide-transformer)** â€“ â­ 801 â€“ Foundation Models for Genomics & Transcriptomics
- **[igvteam/igv](https://github.com/igvteam/igv)** â€“ â­ 712 â€“ Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** â€“ â­ 704 â€“ Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** â€“ â­ 643 â€“ Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** â€“ â­ 641 â€“ Official home of genome aligner based upon notion of Cactus graphs
- **[oschwengers/bakta](https://github.com/oschwengers/bakta)** â€“ â­ 591 â€“ Rapid & standardized annotation of bacterial genomes, MAGs & plasmids
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** â€“ â­ 588 â€“ Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk)** â€“ â­ 570 â€“ GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** â€“ â­ 534 â€“ Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** â€“ â­ 521 â€“ The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[pachterlab/BI-BE-CS-183-2023](https://github.com/pachterlab/BI-BE-CS-183-2023)** â€“ â­ 514 â€“ Introduction to Computational Biology and Bioinformatics Course at Caltech, 2023
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** â€“ â­ 504 â€“ Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[bactopia/bactopia](https://github.com/bactopia/bactopia)** â€“ â­ 494 â€“ A flexible pipeline for complete analysis of bacterial genomes
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** â€“ â­ 481 â€“ PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[evanpeikon/Bioinformatics_Toolkit](https://github.com/evanpeikon/Bioinformatics_Toolkit)** â€“ â­ 456 â€“ A growing collection of bioinformatics tutorials, project guides, tools, and articles Iâ€™ve developed to make bioinformatics more accessible.
- **[MAGICS-LAB/DNABERT_2](https://github.com/MAGICS-LAB/DNABERT_2)** â€“ â­ 447 â€“ [ICLR 2024] DNABERT-2: Efficient Foundation Model and Benchmark for Multi-Species Genome
- **[cov-lineages/pangolin](https://github.com/cov-lineages/pangolin)** â€“ â­ 446 â€“ Software package for assigning SARS-CoV-2 genome sequences to global lineages.
- **[zeqianli/tgv](https://github.com/zeqianli/tgv)** â€“ â­ 429 â€“ Explore genomes in the terminal. Light, blazing fast ğŸš€, vim-motion.
- **[whatshap/whatshap](https://github.com/whatshap/whatshap)** â€“ â­ 401 â€“ Read-based phasing of genomic variants, also called haplotype assembly
- **[lh3/miniprot](https://github.com/lh3/miniprot)** â€“ â­ 394 â€“ Align proteins to genomes with splicing and frameshift
- **[imteekay/machine-learning-research](https://github.com/imteekay/machine-learning-research)** â€“ â­ 384 â€“ âœ¨ ML for Biology & Medicine Research
- **[nextgenusfs/funannotate](https://github.com/nextgenusfs/funannotate)** â€“ â­ 375 â€“ Eukaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** â€“ â­ 362 â€“ Genome-scale protein function classification
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** â€“ â­ 350 â€“ Tools for working with genomic and high throughput sequencing data.
- **[higlass/higlass](https://github.com/higlass/higlass)** â€“ â­ 338 â€“ Fast, flexible and extensible genome browser.
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** â€“ â­ 330 â€“ Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** â€“ â­ 320 â€“ :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** â€“ â­ 318 â€“ Genomic Pre-trained Network
- **[WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM)** â€“ â­ 314 â€“ Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** â€“ â­ 291 â€“ Build a partitioned pangenome graph from microbial genomes
- **[mothur/mothur](https://github.com/mothur/mothur)** â€“ â­ 275 â€“ Welcome to the mothur project, initiated by Dr. Patrick Schloss and his software development team in the Department of Microbiology & Immunology at The University of Michigan. This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.
- **[GMOD/jbrowse-components](https://github.com/GMOD/jbrowse-components)** â€“ â­ 271 â€“ Source code for JBrowse 2, a modern React-based genome browser
- **[hartwigmedical/hmftools](https://github.com/hartwigmedical/hmftools)** â€“ â­ 262 â€“ Various algorithms for analysing genomics data
- **[sanger-pathogens/Artemis](https://github.com/sanger-pathogens/Artemis)** â€“ â­ 261 â€“ Artemis is a free genome viewer and annotation tool that allows visualization of sequence features and the results of analyses within the context of the sequence, and its six-frame translation
- **[biowasm/biowasm](https://github.com/biowasm/biowasm)** â€“ â­ 259 â€“ WebAssembly modules for genomics
- **[ugeneunipro/ugene](https://github.com/ugeneunipro/ugene)** â€“ â­ 255 â€“ UGENE is free open-source cross-platform bioinformatics software
