## Trending Bioinformatics Repositories (Updated: 2025-10-24)

**Showing 55 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** – ⭐ 6273 – Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[satijalab/seurat](https://github.com/satijalab/seurat)** – ⭐ 2544 – R toolkit for single cell genomics
- **[lh3/minimap2](https://github.com/lh3/minimap2)** – ⭐ 2055 – A versatile pairwise aligner for genomic and spliced nucleotide sequences
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** – ⭐ 1737 – Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** – ⭐ 1547 – A library of free open source icons for science illustrations in biology and chemistry
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** – ⭐ 1372 – Aggregate results from bioinformatics analyses across many samples into a single report.
- **[vgteam/vg](https://github.com/vgteam/vg)** – ⭐ 1247 – tools for working with genome variation graphs
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** – ⭐ 1067 – Collection of bioinformatics training materials
- **[pachterlab/gget](https://github.com/pachterlab/gget)** – ⭐ 1062 – 🧬 gget enables efficient querying of genomic reference databases
- **[hail-is/hail](https://github.com/hail-is/hail)** – ⭐ 1032 – Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** – ⭐ 1015 – A list of interesting genome browser and genome visualization programs
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** – ⭐ 982 – scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** – ⭐ 876 – A comprehensive library for computational molecular biology
- **[ablab/spades](https://github.com/ablab/spades)** – ⭐ 873 – SPAdes Genome Assembler
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** – ⭐ 871 – cBioPortal for Cancer Genomics
- **[BioContainers/containers](https://github.com/BioContainers/containers)** – ⭐ 758 – Bioinformatics containers
- **[shenwei356/awesome](https://github.com/shenwei356/awesome)** – ⭐ 712 – Awesome resources on Bioinformatics, data science, machine learning, programming language (Python, Golang, R, Perl) and miscellaneous stuff.
- **[igvteam/igv](https://github.com/igvteam/igv)** – ⭐ 699 – Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[thackl/gggenomes](https://github.com/thackl/gggenomes)** – ⭐ 699 – A grammar of graphics for comparative genomics
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** – ⭐ 690 – Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** – ⭐ 627 – Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** – ⭐ 618 – Official home of genome aligner based upon notion of Cactus graphs
- **[vandijklab/cell2sentence](https://github.com/vandijklab/cell2sentence)** – ⭐ 615 – Cell2Sentence: Teaching Large Language Models the Language of Biology
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** – ⭐ 566 – Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[oschwengers/bakta](https://github.com/oschwengers/bakta)** – ⭐ 561 – Rapid & standardized annotation of bacterial genomes, MAGs & plasmids
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** – ⭐ 524 – Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[lindenb/jvarkit](https://github.com/lindenb/jvarkit)** – ⭐ 510 – Java utilities for Bioinformatics
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** – ⭐ 508 – The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** – ⭐ 498 – Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[bactopia/bactopia](https://github.com/bactopia/bactopia)** – ⭐ 480 – A flexible pipeline for complete analysis of bacterial genomes
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** – ⭐ 470 – PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[cov-lineages/pangolin](https://github.com/cov-lineages/pangolin)** – ⭐ 444 – Software package for assigning SARS-CoV-2 genome sequences to global lineages.
- **[GenerTeam/GENERator](https://github.com/GenerTeam/GENERator)** – ⭐ 434 – GENERator: A Long-Context Generative Genomic Foundation Model
- **[zeqianli/tgv](https://github.com/zeqianli/tgv)** – ⭐ 407 – Explore genomes in the terminal. Light, blazing fast 🚀, vim-motion.
- **[vanheeringen-lab/genomepy](https://github.com/vanheeringen-lab/genomepy)** – ⭐ 402 – genes and genomes at your fingertips
- **[theislab/scarches](https://github.com/theislab/scarches)** – ⭐ 385 – Reference mapping for single-cell genomics
- **[moshi4/pyGenomeViz](https://github.com/moshi4/pyGenomeViz)** – ⭐ 373 – A genome visualization python package for comparative genomics
- **[ncbi/pgap](https://github.com/ncbi/pgap)** – ⭐ 357 – NCBI Prokaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** – ⭐ 351 – Genome-scale protein function classification
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** – ⭐ 348 – Tools for working with genomic and high throughput sequencing data.
- **[higlass/higlass](https://github.com/higlass/higlass)** – ⭐ 334 – Fast, flexible and extensible genome browser.
- **[genometools/genometools](https://github.com/genometools/genometools)** – ⭐ 326 – GenomeTools genome analysis system.
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** – ⭐ 320 – :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** – ⭐ 320 – Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[GenerTeam/GENERanno](https://github.com/GenerTeam/GENERanno)** – ⭐ 301 – GENERanno: A Genomic Foundation Model for Metagenomic Annotation
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** – ⭐ 298 – Genomic Pre-trained Network
- **[dzerbino/velvet](https://github.com/dzerbino/velvet)** – ⭐ 293 – Short read de novo assembler using de Bruijn graphs, as published in: D.R. Zerbino and E. Birney. 2008. Velvet: algorithms for de novo short read assembly using de Bruijn graphs. Genome Research, 18: 821-829
- **[projectglow/glow](https://github.com/projectglow/glow)** – ⭐ 289 – An open-source toolkit for large-scale genomic analysis
- **[wurmlab/sequenceserver](https://github.com/wurmlab/sequenceserver)** – ⭐ 288 – Intuitive graphical web interface for running BLAST bioinformatics tool (i.e. have your own custom NCBI BLAST site!)
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** – ⭐ 284 – Build a partitioned pangenome graph from microbial genomes
- **[tjiangHIT/cuteSV](https://github.com/tjiangHIT/cuteSV)** – ⭐ 273 – Long read based human genomic structural variation detection with cuteSV
- **[mothur/mothur](https://github.com/mothur/mothur)** – ⭐ 273 – Welcome to the mothur project, initiated by Dr. Patrick Schloss and his software development team in the Department of Microbiology & Immunology at The University of Michigan. This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.
- **[sigven/pcgr](https://github.com/sigven/pcgr)** – ⭐ 272 – Personal Cancer Genome Reporter (PCGR)
- **[GMOD/jbrowse-components](https://github.com/GMOD/jbrowse-components)** – ⭐ 259 – Source code for JBrowse 2, a modern React-based genome browser
- **[bluenote-1577/sylph](https://github.com/bluenote-1577/sylph)** – ⭐ 257 – ultrafast taxonomic profiling and genome querying for metagenomic samples by abundance-corrected minhash.
