## Trending Bioinformatics Repositories (Updated: 2025-07-22)

**Showing 74 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** â€“ â­ 6111 â€“ Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[ArcInstitute/evo2](https://github.com/ArcInstitute/evo2)** â€“ â­ 2977 â€“ Genome modeling and design across all domains of life
- **[satijalab/seurat](https://github.com/satijalab/seurat)** â€“ â­ 2476 â€“ R toolkit for single cell genomics
- **[lh3/minimap2](https://github.com/lh3/minimap2)** â€“ â­ 1998 â€“ A versatile pairwise aligner for genomic and spliced nucleotide sequences
- **[rust-bio/rust-bio](https://github.com/rust-bio/rust-bio)** â€“ â­ 1719 â€“ This library provides implementations of many algorithms and data structures that are useful for bioinformatics. All provided implementations are rigorously tested via continuous integration.
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** â€“ â­ 1685 â€“ Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** â€“ â­ 1499 â€“ A library of free open source icons for science illustrations in biology and chemistry
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** â€“ â­ 1329 â€“ Aggregate results from bioinformatics analyses across many samples into a single report.
- **[vgteam/vg](https://github.com/vgteam/vg)** â€“ â­ 1219 â€“ tools for working with genome variation graphs
- **[thieu1995/mealpy](https://github.com/thieu1995/mealpy)** â€“ â­ 1052 â€“ A Collection Of The State-of-the-art Metaheuristic Algorithms In Python (Metaheuristic/Optimizer/Nature-inspired/Biology)
- **[pachterlab/gget](https://github.com/pachterlab/gget)** â€“ â­ 1043 â€“ ğŸ§¬ gget enables efficient querying of genomic reference databases
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** â€“ â­ 1036 â€“ Collection of bioinformatics training materials
- **[bigdatagenomics/adam](https://github.com/bigdatagenomics/adam)** â€“ â­ 1028 â€“ ADAM is a genomics analysis platform with specialized file formats built using Apache Avro, Apache Spark, and Apache Parquet. Apache 2 licensed.
- **[hail-is/hail](https://github.com/hail-is/hail)** â€“ â­ 1018 â€“ Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** â€“ â­ 982 â€“ A list of interesting genome browser and genome visualization programs
- **[marbl/CHM13](https://github.com/marbl/CHM13)** â€“ â­ 970 â€“ The complete sequence of a human genome
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** â€“ â­ 952 â€“ scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[ablab/spades](https://github.com/ablab/spades)** â€“ â­ 852 â€“ SPAdes Genome Assembler
- **[pysam-developers/pysam](https://github.com/pysam-developers/pysam)** â€“ â­ 838 â€“ Pysam is a Python package for reading, manipulating, and writing genomics data such as SAM/BAM/CRAM and VCF/BCF files. It's a lightweight wrapper of the HTSlib API, the same one that powers samtools, bcftools, and tabix.
- **[tanghaibao/jcvi](https://github.com/tanghaibao/jcvi)** â€“ â­ 836 â€“ Python library to facilitate genome assembly, annotation, and comparative genomics
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** â€“ â­ 828 â€“ A comprehensive library for computational molecular biology
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** â€“ â­ 826 â€“ cBioPortal for Cancer Genomics
- **[davidemms/OrthoFinder](https://github.com/davidemms/OrthoFinder)** â€“ â­ 787 â€“ Phylogenetic orthology inference for comparative genomics
- **[thackl/gggenomes](https://github.com/thackl/gggenomes)** â€“ â­ 688 â€“ A grammar of graphics for comparative genomics
- **[igvteam/igv](https://github.com/igvteam/igv)** â€“ â­ 686 â€“ Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[marbl/canu](https://github.com/marbl/canu)** â€“ â­ 684 â€“ A single molecule sequence assembler for genomes large and small.
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** â€“ â­ 680 â€“ Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[jerryji1993/DNABERT](https://github.com/jerryji1993/DNABERT)** â€“ â­ 680 â€“ DNABERT: pre-trained Bidirectional Encoder Representations from Transformers model for DNA-language in genome
- **[instadeepai/nucleotide-transformer](https://github.com/instadeepai/nucleotide-transformer)** â€“ â­ 661 â€“ Foundation Models for Genomics & Transcriptomics
- **[eggnogdb/eggnog-mapper](https://github.com/eggnogdb/eggnog-mapper)** â€“ â­ 633 â€“ Fast genome-wide functional annotation through orthology assignment
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** â€“ â­ 600 â€“ Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** â€“ â­ 592 â€“ Official home of genome aligner based upon notion of Cactus graphs
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** â€“ â­ 539 â€“ Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[oschwengers/bakta](https://github.com/oschwengers/bakta)** â€“ â­ 532 â€“ Rapid & standardized annotation of bacterial genomes, MAGs & plasmids
- **[Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk)** â€“ â­ 532 â€“ GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** â€“ â­ 513 â€“ Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[lindenb/jvarkit](https://github.com/lindenb/jvarkit)** â€“ â­ 504 â€“ Java utilities for Bioinformatics
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** â€“ â­ 495 â€“ The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** â€“ â­ 492 â€“ Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[GMOD/jbrowse](https://github.com/GMOD/jbrowse)** â€“ â­ 471 â€“ JBrowse 1, a full-featured genome browser built with JavaScript and HTML5. For JBrowse 2, see https://github.com/GMOD/jbrowse-components.
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** â€“ â­ 459 â€“ PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[cov-lineages/pangolin](https://github.com/cov-lineages/pangolin)** â€“ â­ 439 â€“ Software package for assigning SARS-CoV-2 genome sequences to global lineages.
- **[GenerTeam/GENERator](https://github.com/GenerTeam/GENERator)** â€“ â­ 426 â€“ GENERator: A Long-Context Generative Genomic Foundation Model
- **[zeqianli/tgv](https://github.com/zeqianli/tgv)** â€“ â­ 379 â€“ Explore genomes in the terminal. Light, blazing fast ğŸš€, vim-motion.
- **[whatshap/whatshap](https://github.com/whatshap/whatshap)** â€“ â­ 376 â€“ Read-based phasing of genomic variants, also called haplotype assembly
- **[Ecogenomics/CheckM](https://github.com/Ecogenomics/CheckM)** â€“ â­ 376 â€“ Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes
- **[lh3/miniprot](https://github.com/lh3/miniprot)** â€“ â­ 375 â€“ Align proteins to genomes with splicing and frameshift
- **[openproblems-bio/openproblems](https://github.com/openproblems-bio/openproblems)** â€“ â­ 374 â€“ Formalizing and benchmarking open problems in single-cell genomics
- **[genetics-statistics/GEMMA](https://github.com/genetics-statistics/GEMMA)** â€“ â­ 366 â€“ Genome-wide Efficient Mixed Model Association
- **[kcleal/gw](https://github.com/kcleal/gw)** â€“ â­ 365 â€“ Genome browser and variant annotation
- **[nextgenusfs/funannotate](https://github.com/nextgenusfs/funannotate)** â€“ â­ 357 â€“ Eukaryotic Genome Annotation Pipeline
- **[ncbi/pgap](https://github.com/ncbi/pgap)** â€“ â­ 343 â€“ NCBI Prokaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** â€“ â­ 338 â€“ Genome-scale protein function classification
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** â€“ â­ 336 â€“ Tools for working with genomic and high throughput sequencing data.
- **[higlass/higlass](https://github.com/higlass/higlass)** â€“ â­ 328 â€“ Fast, flexible and extensible genome browser.
- **[bernatgel/karyoploteR](https://github.com/bernatgel/karyoploteR)** â€“ â­ 325 â€“ karyoploteR - An R/Bioconductor package to plot arbitrary data along the genome
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** â€“ â­ 316 â€“ :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[Gaius-Augustus/Augustus](https://github.com/Gaius-Augustus/Augustus)** â€“ â­ 313 â€“ Genome annotation with AUGUSTUS
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** â€“ â­ 311 â€“ Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[MrOlm/drep](https://github.com/MrOlm/drep)** â€“ â­ 302 â€“ Rapid comparison and dereplication of genomes
- **[xiaolei-lab/rMVP](https://github.com/xiaolei-lab/rMVP)** â€“ â­ 302 â€“ :postbox: A Memory-efficient, Visualization-enhanced, and Parallel-accelerated Tool For Genome-Wide Association Study
- **[CompEpigen/figeno](https://github.com/CompEpigen/figeno)** â€“ â­ 296 â€“ Tool for plotting sequencing data along genomic coordinates.
- **[WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM)** â€“ â­ 288 â€“ Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes
- **[wurmlab/sequenceserver](https://github.com/wurmlab/sequenceserver)** â€“ â­ 286 â€“ Intuitive graphical web interface for running BLAST bioinformatics tool (i.e. have your own custom NCBI BLAST site!)
- **[MariaNattestad/Ribbon](https://github.com/MariaNattestad/Ribbon)** â€“ â­ 285 â€“ A genome browser designed for complex structural variants and long reads.
- **[evanpeikon/Bioinformatics_Toolkit](https://github.com/evanpeikon/Bioinformatics_Toolkit)** â€“ â­ 284 â€“ A growing collection of bioinformatics tutorials, project guides, tools, and articles Iâ€™ve developed to make bioinformatics more accessible.
- **[projectglow/glow](https://github.com/projectglow/glow)** â€“ â­ 284 â€“ An open-source toolkit for large-scale genomic analysis
- **[GenerTeam/GENERanno](https://github.com/GenerTeam/GENERanno)** â€“ â­ 280 â€“ GENERanno: A Genomic Foundation Model for Metagenomic Annotation
- **[KamilSJaron/smudgeplot](https://github.com/KamilSJaron/smudgeplot)** â€“ â­ 274 â€“ Inference of ploidy and heterozygosity structure using whole genome sequencing data
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** â€“ â­ 273 â€“ Genomic Pre-trained Network
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** â€“ â­ 273 â€“ Build a partitioned pangenome graph from microbial genomes
- **[lynnlangit/gcp-for-bioinformatics](https://github.com/lynnlangit/gcp-for-bioinformatics)** â€“ â­ 268 â€“ GCP for Bioinformatics Researchers
- **[sigven/pcgr](https://github.com/sigven/pcgr)** â€“ â­ 268 â€“ Personal Cancer Genome Reporter (PCGR)
- **[biowasm/biowasm](https://github.com/biowasm/biowasm)** â€“ â­ 252 â€“ WebAssembly modules for genomics
