## Trending Bioinformatics Repositories (Updated: 2025-08-21)

**Showing 57 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** – ⭐ 6154 – Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[ArcInstitute/evo2](https://github.com/ArcInstitute/evo2)** – ⭐ 3031 – Genome modeling and design across all domains of life
- **[satijalab/seurat](https://github.com/satijalab/seurat)** – ⭐ 2495 – R toolkit for single cell genomics
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** – ⭐ 1704 – Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** – ⭐ 1513 – A library of free open source icons for science illustrations in biology and chemistry
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** – ⭐ 1344 – Aggregate results from bioinformatics analyses across many samples into a single report.
- **[crazyhottommy/getting-started-with-genomics-tools-and-resources](https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources)** – ⭐ 1306 – Unix, R and python tools for genomics and data science
- **[vgteam/vg](https://github.com/vgteam/vg)** – ⭐ 1225 – tools for working with genome variation graphs
- **[thieu1995/mealpy](https://github.com/thieu1995/mealpy)** – ⭐ 1069 – A Collection Of The State-of-the-art Metaheuristic Algorithms In Python (Metaheuristic/Optimizer/Nature-inspired/Biology)
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** – ⭐ 1050 – Collection of bioinformatics training materials
- **[pachterlab/gget](https://github.com/pachterlab/gget)** – ⭐ 1047 – 🧬 gget enables efficient querying of genomic reference databases
- **[kblin/ncbi-genome-download](https://github.com/kblin/ncbi-genome-download)** – ⭐ 1026 – Scripts to download genomes from the NCBI FTP servers
- **[hail-is/hail](https://github.com/hail-is/hail)** – ⭐ 1025 – Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** – ⭐ 998 – A list of interesting genome browser and genome visualization programs
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** – ⭐ 960 – scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[ablab/spades](https://github.com/ablab/spades)** – ⭐ 860 – SPAdes Genome Assembler
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** – ⭐ 848 – A comprehensive library for computational molecular biology
- **[tanghaibao/jcvi](https://github.com/tanghaibao/jcvi)** – ⭐ 844 – Python library to facilitate genome assembly, annotation, and comparative genomics
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** – ⭐ 842 – cBioPortal for Cancer Genomics
- **[shenwei356/awesome](https://github.com/shenwei356/awesome)** – ⭐ 699 – Awesome resources on Bioinformatics, data science, machine learning, programming language (Python, Golang, R, Perl) and miscellaneous stuff.
- **[igvteam/igv](https://github.com/igvteam/igv)** – ⭐ 695 – Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[marbl/canu](https://github.com/marbl/canu)** – ⭐ 688 – A single molecule sequence assembler for genomes large and small.
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** – ⭐ 684 – Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** – ⭐ 609 – Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** – ⭐ 598 – Official home of genome aligner based upon notion of Cactus graphs
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** – ⭐ 548 – Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[jorvis/biocode](https://github.com/jorvis/biocode)** – ⭐ 541 – Bioinformatics code libraries and scripts
- **[Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk)** – ⭐ 538 – GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** – ⭐ 519 – Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[lindenb/jvarkit](https://github.com/lindenb/jvarkit)** – ⭐ 510 – Java utilities for Bioinformatics
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** – ⭐ 501 – The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** – ⭐ 493 – Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[bactopia/bactopia](https://github.com/bactopia/bactopia)** – ⭐ 468 – A flexible pipeline for complete analysis of bacterial genomes
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** – ⭐ 462 – PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[GenerTeam/GENERator](https://github.com/GenerTeam/GENERator)** – ⭐ 430 – GENERator: A Long-Context Generative Genomic Foundation Model
- **[ParBLiSS/FastANI](https://github.com/ParBLiSS/FastANI)** – ⭐ 426 – Fast Whole-Genome Similarity (ANI) Estimation
- **[MAGICS-LAB/DNABERT_2](https://github.com/MAGICS-LAB/DNABERT_2)** – ⭐ 408 – [ICLR 2024] DNABERT-2: Efficient Foundation Model and Benchmark for Multi-Species Genome
- **[openvax/pyensembl](https://github.com/openvax/pyensembl)** – ⭐ 398 – Python interface to access reference genome features (such as genes, transcripts, and exons) from Ensembl
- **[zeqianli/tgv](https://github.com/zeqianli/tgv)** – ⭐ 394 – Explore genomes in the terminal. Light, blazing fast 🚀, vim-motion.
- **[openproblems-bio/openproblems](https://github.com/openproblems-bio/openproblems)** – ⭐ 383 – Formalizing and benchmarking open problems in single-cell genomics
- **[theislab/scarches](https://github.com/theislab/scarches)** – ⭐ 381 – Reference mapping for single-cell genomics
- **[kcleal/gw](https://github.com/kcleal/gw)** – ⭐ 369 – Genome browser and variant annotation
- **[ncbi/pgap](https://github.com/ncbi/pgap)** – ⭐ 350 – NCBI Prokaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** – ⭐ 341 – Genome-scale protein function classification
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** – ⭐ 341 – Tools for working with genomic and high throughput sequencing data.
- **[higlass/higlass](https://github.com/higlass/higlass)** – ⭐ 332 – Fast, flexible and extensible genome browser.
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** – ⭐ 317 – :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** – ⭐ 315 – Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[Gaius-Augustus/Augustus](https://github.com/Gaius-Augustus/Augustus)** – ⭐ 314 – Genome annotation with AUGUSTUS
- **[GenerTeam/GENERanno](https://github.com/GenerTeam/GENERanno)** – ⭐ 300 – GENERanno: A Genomic Foundation Model for Metagenomic Annotation
- **[WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM)** – ⭐ 290 – Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes
- **[wurmlab/sequenceserver](https://github.com/wurmlab/sequenceserver)** – ⭐ 287 – Intuitive graphical web interface for running BLAST bioinformatics tool (i.e. have your own custom NCBI BLAST site!)
- **[projectglow/glow](https://github.com/projectglow/glow)** – ⭐ 287 – An open-source toolkit for large-scale genomic analysis
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** – ⭐ 276 – Genomic Pre-trained Network
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** – ⭐ 275 – Build a partitioned pangenome graph from microbial genomes
- **[mothur/mothur](https://github.com/mothur/mothur)** – ⭐ 272 – Welcome to the mothur project, initiated by Dr. Patrick Schloss and his software development team in the Department of Microbiology & Immunology at The University of Michigan. This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.
- **[GMOD/jbrowse-components](https://github.com/GMOD/jbrowse-components)** – ⭐ 251 – Source code for JBrowse 2, a modern React-based genome browser
