## Trending Bioinformatics Repositories (Updated: 2025-08-21)

**Showing 57 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** â€“ â­ 6154 â€“ Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[ArcInstitute/evo2](https://github.com/ArcInstitute/evo2)** â€“ â­ 3031 â€“ Genome modeling and design across all domains of life
- **[satijalab/seurat](https://github.com/satijalab/seurat)** â€“ â­ 2495 â€“ R toolkit for single cell genomics
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** â€“ â­ 1704 â€“ Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** â€“ â­ 1513 â€“ A library of free open source icons for science illustrations in biology and chemistry
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** â€“ â­ 1344 â€“ Aggregate results from bioinformatics analyses across many samples into a single report.
- **[crazyhottommy/getting-started-with-genomics-tools-and-resources](https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources)** â€“ â­ 1306 â€“ Unix, R and python tools for genomics and data science
- **[vgteam/vg](https://github.com/vgteam/vg)** â€“ â­ 1225 â€“ tools for working with genome variation graphs
- **[thieu1995/mealpy](https://github.com/thieu1995/mealpy)** â€“ â­ 1069 â€“ A Collection Of The State-of-the-art Metaheuristic Algorithms In Python (Metaheuristic/Optimizer/Nature-inspired/Biology)
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** â€“ â­ 1050 â€“ Collection of bioinformatics training materials
- **[pachterlab/gget](https://github.com/pachterlab/gget)** â€“ â­ 1047 â€“ ğŸ§¬ gget enables efficient querying of genomic reference databases
- **[kblin/ncbi-genome-download](https://github.com/kblin/ncbi-genome-download)** â€“ â­ 1026 â€“ Scripts to download genomes from the NCBI FTP servers
- **[hail-is/hail](https://github.com/hail-is/hail)** â€“ â­ 1025 â€“ Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** â€“ â­ 998 â€“ A list of interesting genome browser and genome visualization programs
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** â€“ â­ 960 â€“ scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[ablab/spades](https://github.com/ablab/spades)** â€“ â­ 860 â€“ SPAdes Genome Assembler
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** â€“ â­ 848 â€“ A comprehensive library for computational molecular biology
- **[tanghaibao/jcvi](https://github.com/tanghaibao/jcvi)** â€“ â­ 844 â€“ Python library to facilitate genome assembly, annotation, and comparative genomics
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** â€“ â­ 842 â€“ cBioPortal for Cancer Genomics
- **[shenwei356/awesome](https://github.com/shenwei356/awesome)** â€“ â­ 699 â€“ Awesome resources on Bioinformatics, data science, machine learning, programming language (Python, Golang, R, Perl) and miscellaneous stuff.
- **[igvteam/igv](https://github.com/igvteam/igv)** â€“ â­ 695 â€“ Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[marbl/canu](https://github.com/marbl/canu)** â€“ â­ 688 â€“ A single molecule sequence assembler for genomes large and small.
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** â€“ â­ 684 â€“ Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** â€“ â­ 609 â€“ Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** â€“ â­ 598 â€“ Official home of genome aligner based upon notion of Cactus graphs
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** â€“ â­ 548 â€“ Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[jorvis/biocode](https://github.com/jorvis/biocode)** â€“ â­ 541 â€“ Bioinformatics code libraries and scripts
- **[Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk)** â€“ â­ 538 â€“ GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** â€“ â­ 519 â€“ Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[lindenb/jvarkit](https://github.com/lindenb/jvarkit)** â€“ â­ 510 â€“ Java utilities for Bioinformatics
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** â€“ â­ 501 â€“ The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** â€“ â­ 493 â€“ Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[bactopia/bactopia](https://github.com/bactopia/bactopia)** â€“ â­ 468 â€“ A flexible pipeline for complete analysis of bacterial genomes
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** â€“ â­ 462 â€“ PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[GenerTeam/GENERator](https://github.com/GenerTeam/GENERator)** â€“ â­ 430 â€“ GENERator: A Long-Context Generative Genomic Foundation Model
- **[ParBLiSS/FastANI](https://github.com/ParBLiSS/FastANI)** â€“ â­ 426 â€“ Fast Whole-Genome Similarity (ANI) Estimation
- **[MAGICS-LAB/DNABERT_2](https://github.com/MAGICS-LAB/DNABERT_2)** â€“ â­ 408 â€“ [ICLR 2024] DNABERT-2: Efficient Foundation Model and Benchmark for Multi-Species Genome
- **[openvax/pyensembl](https://github.com/openvax/pyensembl)** â€“ â­ 398 â€“ Python interface to access reference genome features (such as genes, transcripts, and exons) from Ensembl
- **[zeqianli/tgv](https://github.com/zeqianli/tgv)** â€“ â­ 394 â€“ Explore genomes in the terminal. Light, blazing fast ğŸš€, vim-motion.
- **[openproblems-bio/openproblems](https://github.com/openproblems-bio/openproblems)** â€“ â­ 383 â€“ Formalizing and benchmarking open problems in single-cell genomics
- **[theislab/scarches](https://github.com/theislab/scarches)** â€“ â­ 381 â€“ Reference mapping for single-cell genomics
- **[kcleal/gw](https://github.com/kcleal/gw)** â€“ â­ 369 â€“ Genome browser and variant annotation
- **[ncbi/pgap](https://github.com/ncbi/pgap)** â€“ â­ 350 â€“ NCBI Prokaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** â€“ â­ 341 â€“ Genome-scale protein function classification
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** â€“ â­ 341 â€“ Tools for working with genomic and high throughput sequencing data.
- **[higlass/higlass](https://github.com/higlass/higlass)** â€“ â­ 332 â€“ Fast, flexible and extensible genome browser.
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** â€“ â­ 317 â€“ :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** â€“ â­ 315 â€“ Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[Gaius-Augustus/Augustus](https://github.com/Gaius-Augustus/Augustus)** â€“ â­ 314 â€“ Genome annotation with AUGUSTUS
- **[GenerTeam/GENERanno](https://github.com/GenerTeam/GENERanno)** â€“ â­ 300 â€“ GENERanno: A Genomic Foundation Model for Metagenomic Annotation
- **[WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM)** â€“ â­ 290 â€“ Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes
- **[wurmlab/sequenceserver](https://github.com/wurmlab/sequenceserver)** â€“ â­ 287 â€“ Intuitive graphical web interface for running BLAST bioinformatics tool (i.e. have your own custom NCBI BLAST site!)
- **[projectglow/glow](https://github.com/projectglow/glow)** â€“ â­ 287 â€“ An open-source toolkit for large-scale genomic analysis
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** â€“ â­ 276 â€“ Genomic Pre-trained Network
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** â€“ â­ 275 â€“ Build a partitioned pangenome graph from microbial genomes
- **[mothur/mothur](https://github.com/mothur/mothur)** â€“ â­ 272 â€“ Welcome to the mothur project, initiated by Dr. Patrick Schloss and his software development team in the Department of Microbiology & Immunology at The University of Michigan. This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.
- **[GMOD/jbrowse-components](https://github.com/GMOD/jbrowse-components)** â€“ â­ 251 â€“ Source code for JBrowse 2, a modern React-based genome browser
