## Trending Bioinformatics Repositories (Updated: 2025-12-09)

**Showing 60 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** â€“ â­ 6387 â€“ Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[satijalab/seurat](https://github.com/satijalab/seurat)** â€“ â­ 2581 â€“ R toolkit for single cell genomics
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** â€“ â­ 1766 â€“ Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** â€“ â­ 1577 â€“ A library of free open source icons for science illustrations in biology and chemistry
- **[evo-design/evo](https://github.com/evo-design/evo)** â€“ â­ 1460 â€“ Biological foundation modeling from molecular to genome scale
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** â€“ â­ 1396 â€“ Aggregate results from bioinformatics analyses across many samples into a single report.
- **[vgteam/vg](https://github.com/vgteam/vg)** â€“ â­ 1267 â€“ tools for working with genome variation graphs
- **[pachterlab/gget](https://github.com/pachterlab/gget)** â€“ â­ 1082 â€“ ğŸ§¬ gget enables efficient querying of genomic reference databases
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** â€“ â­ 1079 â€“ Collection of bioinformatics training materials
- **[hail-is/hail](https://github.com/hail-is/hail)** â€“ â­ 1039 â€“ Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** â€“ â­ 1035 â€“ A list of interesting genome browser and genome visualization programs
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** â€“ â­ 994 â€“ scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** â€“ â­ 899 â€“ cBioPortal for Cancer Genomics
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** â€“ â­ 892 â€“ A comprehensive library for computational molecular biology
- **[ablab/spades](https://github.com/ablab/spades)** â€“ â­ 883 â€“ SPAdes Genome Assembler
- **[tanghaibao/jcvi](https://github.com/tanghaibao/jcvi)** â€“ â­ 878 â€“ Python library to facilitate genome assembly, annotation, and comparative genomics
- **[pysam-developers/pysam](https://github.com/pysam-developers/pysam)** â€“ â­ 871 â€“ Pysam is a Python package for reading, manipulating, and writing genomics data such as SAM/BAM/CRAM and VCF/BCF files. It's a lightweight wrapper of the HTSlib API, the same one that powers samtools, bcftools, and tabix.
- **[shenwei356/awesome](https://github.com/shenwei356/awesome)** â€“ â­ 726 â€“ Awesome resources on Bioinformatics, data science, machine learning, programming language (Python, Golang, R, Perl) and miscellaneous stuff.
- **[thackl/gggenomes](https://github.com/thackl/gggenomes)** â€“ â­ 723 â€“ A grammar of graphics for comparative genomics
- **[igvteam/igv](https://github.com/igvteam/igv)** â€“ â­ 706 â€“ Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** â€“ â­ 701 â€“ Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** â€“ â­ 635 â€“ Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** â€“ â­ 630 â€“ Official home of genome aligner based upon notion of Cactus graphs
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** â€“ â­ 578 â€“ Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[oschwengers/bakta](https://github.com/oschwengers/bakta)** â€“ â­ 570 â€“ Rapid & standardized annotation of bacterial genomes, MAGs & plasmids
- **[tseemann/snippy](https://github.com/tseemann/snippy)** â€“ â­ 555 â€“ :scissors: :zap: Rapid haploid variant calling and core genome alignment
- **[Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk)** â€“ â­ 555 â€“ GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** â€“ â­ 530 â€“ Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[lindenb/jvarkit](https://github.com/lindenb/jvarkit)** â€“ â­ 513 â€“ Java utilities for Bioinformatics
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** â€“ â­ 513 â€“ The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** â€“ â­ 503 â€“ Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[bactopia/bactopia](https://github.com/bactopia/bactopia)** â€“ â­ 488 â€“ A flexible pipeline for complete analysis of bacterial genomes
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** â€“ â­ 475 â€“ PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[cov-lineages/pangolin](https://github.com/cov-lineages/pangolin)** â€“ â­ 442 â€“ Software package for assigning SARS-CoV-2 genome sequences to global lineages.
- **[GenerTeam/GENERator](https://github.com/GenerTeam/GENERator)** â€“ â­ 439 â€“ GENERator: A Long-Context Generative Genomic Foundation Model
- **[zeqianli/tgv](https://github.com/zeqianli/tgv)** â€“ â­ 424 â€“ Explore genomes in the terminal. Light, blazing fast ğŸš€, vim-motion.
- **[openproblems-bio/openproblems](https://github.com/openproblems-bio/openproblems)** â€“ â­ 413 â€“ Formalizing and benchmarking open problems in single-cell genomics
- **[theislab/scarches](https://github.com/theislab/scarches)** â€“ â­ 394 â€“ Reference mapping for single-cell genomics
- **[genetics-statistics/GEMMA](https://github.com/genetics-statistics/GEMMA)** â€“ â­ 385 â€“ Genome-wide Efficient Mixed Model Association
- **[moshi4/pyGenomeViz](https://github.com/moshi4/pyGenomeViz)** â€“ â­ 379 â€“ A genome visualization python package for comparative genomics
- **[ncbi/pgap](https://github.com/ncbi/pgap)** â€“ â­ 362 â€“ NCBI Prokaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** â€“ â­ 354 â€“ Genome-scale protein function classification
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** â€“ â­ 349 â€“ Tools for working with genomic and high throughput sequencing data.
- **[genometools/genometools](https://github.com/genometools/genometools)** â€“ â­ 330 â€“ GenomeTools genome analysis system.
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** â€“ â­ 328 â€“ Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[MrOlm/drep](https://github.com/MrOlm/drep)** â€“ â­ 321 â€“ Rapid comparison and dereplication of genomes
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** â€“ â­ 320 â€“ :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** â€“ â­ 312 â€“ Genomic Pre-trained Network
- **[WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM)** â€“ â­ 303 â€“ Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes
- **[GenerTeam/GENERanno](https://github.com/GenerTeam/GENERanno)** â€“ â­ 302 â€“ GENERanno: A Genomic Foundation Model for Metagenomic Annotation
- **[projectglow/glow](https://github.com/projectglow/glow)** â€“ â­ 292 â€“ An open-source toolkit for large-scale genomic analysis
- **[wurmlab/sequenceserver](https://github.com/wurmlab/sequenceserver)** â€“ â­ 289 â€“ Intuitive graphical web interface for running BLAST bioinformatics tool (i.e. have your own custom NCBI BLAST site!)
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** â€“ â­ 288 â€“ Build a partitioned pangenome graph from microbial genomes
- **[KamilSJaron/smudgeplot](https://github.com/KamilSJaron/smudgeplot)** â€“ â­ 285 â€“ Inference of ploidy and heterozygosity structure using whole genome sequencing data
- **[ai-boost/awesome-ai-for-science](https://github.com/ai-boost/awesome-ai-for-science)** â€“ â­ 282 â€“ A curated list of awesome AI tools, libraries, papers, datasets, and frameworks that accelerate scientific discovery â€” from physics and chemistry to biology, materials, and beyond.
- **[mothur/mothur](https://github.com/mothur/mothur)** â€“ â­ 274 â€“ Welcome to the mothur project, initiated by Dr. Patrick Schloss and his software development team in the Department of Microbiology & Immunology at The University of Michigan. This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.
- **[GMOD/jbrowse-components](https://github.com/GMOD/jbrowse-components)** â€“ â­ 268 â€“ Source code for JBrowse 2, a modern React-based genome browser
- **[hartwigmedical/hmftools](https://github.com/hartwigmedical/hmftools)** â€“ â­ 255 â€“ Various algorithms for analysing genomics data
- **[biowasm/biowasm](https://github.com/biowasm/biowasm)** â€“ â­ 255 â€“ WebAssembly modules for genomics
- **[ugeneunipro/ugene](https://github.com/ugeneunipro/ugene)** â€“ â­ 253 â€“ UGENE is free open-source cross-platform bioinformatics software
