## Trending Bioinformatics Repositories (Updated: 2025-12-07)

**Showing 57 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** ‚Äì ‚≠ê 6377 ‚Äì Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[satijalab/seurat](https://github.com/satijalab/seurat)** ‚Äì ‚≠ê 2581 ‚Äì R toolkit for single cell genomics
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** ‚Äì ‚≠ê 1765 ‚Äì Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** ‚Äì ‚≠ê 1575 ‚Äì A library of free open source icons for science illustrations in biology and chemistry
- **[evo-design/evo](https://github.com/evo-design/evo)** ‚Äì ‚≠ê 1459 ‚Äì Biological foundation modeling from molecular to genome scale
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** ‚Äì ‚≠ê 1395 ‚Äì Aggregate results from bioinformatics analyses across many samples into a single report.
- **[vgteam/vg](https://github.com/vgteam/vg)** ‚Äì ‚≠ê 1266 ‚Äì tools for working with genome variation graphs
- **[pachterlab/gget](https://github.com/pachterlab/gget)** ‚Äì ‚≠ê 1081 ‚Äì üß¨ gget enables efficient querying of genomic reference databases
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** ‚Äì ‚≠ê 1079 ‚Äì Collection of bioinformatics training materials
- **[hail-is/hail](https://github.com/hail-is/hail)** ‚Äì ‚≠ê 1038 ‚Äì Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** ‚Äì ‚≠ê 1032 ‚Äì A list of interesting genome browser and genome visualization programs
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** ‚Äì ‚≠ê 992 ‚Äì scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** ‚Äì ‚≠ê 895 ‚Äì cBioPortal for Cancer Genomics
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** ‚Äì ‚≠ê 892 ‚Äì A comprehensive library for computational molecular biology
- **[ablab/spades](https://github.com/ablab/spades)** ‚Äì ‚≠ê 881 ‚Äì SPAdes Genome Assembler
- **[tanghaibao/jcvi](https://github.com/tanghaibao/jcvi)** ‚Äì ‚≠ê 877 ‚Äì Python library to facilitate genome assembly, annotation, and comparative genomics
- **[pysam-developers/pysam](https://github.com/pysam-developers/pysam)** ‚Äì ‚≠ê 871 ‚Äì Pysam is a Python package for reading, manipulating, and writing genomics data such as SAM/BAM/CRAM and VCF/BCF files. It's a lightweight wrapper of the HTSlib API, the same one that powers samtools, bcftools, and tabix.
- **[shenwei356/awesome](https://github.com/shenwei356/awesome)** ‚Äì ‚≠ê 726 ‚Äì Awesome resources on Bioinformatics, data science, machine learning, programming language (Python, Golang, R, Perl) and miscellaneous stuff.
- **[thackl/gggenomes](https://github.com/thackl/gggenomes)** ‚Äì ‚≠ê 722 ‚Äì A grammar of graphics for comparative genomics
- **[igvteam/igv](https://github.com/igvteam/igv)** ‚Äì ‚≠ê 706 ‚Äì Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** ‚Äì ‚≠ê 700 ‚Äì Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** ‚Äì ‚≠ê 635 ‚Äì Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** ‚Äì ‚≠ê 630 ‚Äì Official home of genome aligner based upon notion of Cactus graphs
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** ‚Äì ‚≠ê 578 ‚Äì Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[oschwengers/bakta](https://github.com/oschwengers/bakta)** ‚Äì ‚≠ê 570 ‚Äì Rapid & standardized annotation of bacterial genomes, MAGs & plasmids
- **[Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk)** ‚Äì ‚≠ê 555 ‚Äì GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** ‚Äì ‚≠ê 530 ‚Äì Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[lindenb/jvarkit](https://github.com/lindenb/jvarkit)** ‚Äì ‚≠ê 513 ‚Äì Java utilities for Bioinformatics
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** ‚Äì ‚≠ê 513 ‚Äì The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** ‚Äì ‚≠ê 502 ‚Äì Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[bactopia/bactopia](https://github.com/bactopia/bactopia)** ‚Äì ‚≠ê 487 ‚Äì A flexible pipeline for complete analysis of bacterial genomes
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** ‚Äì ‚≠ê 475 ‚Äì PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[cov-lineages/pangolin](https://github.com/cov-lineages/pangolin)** ‚Äì ‚≠ê 443 ‚Äì Software package for assigning SARS-CoV-2 genome sequences to global lineages.
- **[GenerTeam/GENERator](https://github.com/GenerTeam/GENERator)** ‚Äì ‚≠ê 438 ‚Äì GENERator: A Long-Context Generative Genomic Foundation Model
- **[openproblems-bio/openproblems](https://github.com/openproblems-bio/openproblems)** ‚Äì ‚≠ê 412 ‚Äì Formalizing and benchmarking open problems in single-cell genomics
- **[theislab/scarches](https://github.com/theislab/scarches)** ‚Äì ‚≠ê 394 ‚Äì Reference mapping for single-cell genomics
- **[genetics-statistics/GEMMA](https://github.com/genetics-statistics/GEMMA)** ‚Äì ‚≠ê 385 ‚Äì Genome-wide Efficient Mixed Model Association
- **[ncbi/pgap](https://github.com/ncbi/pgap)** ‚Äì ‚≠ê 361 ‚Äì NCBI Prokaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** ‚Äì ‚≠ê 354 ‚Äì Genome-scale protein function classification
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** ‚Äì ‚≠ê 349 ‚Äì Tools for working with genomic and high throughput sequencing data.
- **[genometools/genometools](https://github.com/genometools/genometools)** ‚Äì ‚≠ê 330 ‚Äì GenomeTools genome analysis system.
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** ‚Äì ‚≠ê 327 ‚Äì Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[MrOlm/drep](https://github.com/MrOlm/drep)** ‚Äì ‚≠ê 321 ‚Äì Rapid comparison and dereplication of genomes
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** ‚Äì ‚≠ê 320 ‚Äì :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** ‚Äì ‚≠ê 309 ‚Äì Genomic Pre-trained Network
- **[WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM)** ‚Äì ‚≠ê 303 ‚Äì Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes
- **[GenerTeam/GENERanno](https://github.com/GenerTeam/GENERanno)** ‚Äì ‚≠ê 302 ‚Äì GENERanno: A Genomic Foundation Model for Metagenomic Annotation
- **[projectglow/glow](https://github.com/projectglow/glow)** ‚Äì ‚≠ê 292 ‚Äì An open-source toolkit for large-scale genomic analysis
- **[wurmlab/sequenceserver](https://github.com/wurmlab/sequenceserver)** ‚Äì ‚≠ê 289 ‚Äì Intuitive graphical web interface for running BLAST bioinformatics tool (i.e. have your own custom NCBI BLAST site!)
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** ‚Äì ‚≠ê 287 ‚Äì Build a partitioned pangenome graph from microbial genomes
- **[KamilSJaron/smudgeplot](https://github.com/KamilSJaron/smudgeplot)** ‚Äì ‚≠ê 285 ‚Äì Inference of ploidy and heterozygosity structure using whole genome sequencing data
- **[ai-boost/awesome-ai-for-science](https://github.com/ai-boost/awesome-ai-for-science)** ‚Äì ‚≠ê 281 ‚Äì A curated list of awesome AI tools, libraries, papers, datasets, and frameworks that accelerate scientific discovery ‚Äî from physics and chemistry to biology, materials, and beyond.
- **[mothur/mothur](https://github.com/mothur/mothur)** ‚Äì ‚≠ê 274 ‚Äì Welcome to the mothur project, initiated by Dr. Patrick Schloss and his software development team in the Department of Microbiology & Immunology at The University of Michigan. This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.
- **[GMOD/jbrowse-components](https://github.com/GMOD/jbrowse-components)** ‚Äì ‚≠ê 268 ‚Äì Source code for JBrowse 2, a modern React-based genome browser
- **[hartwigmedical/hmftools](https://github.com/hartwigmedical/hmftools)** ‚Äì ‚≠ê 255 ‚Äì Various algorithms for analysing genomics data
- **[biowasm/biowasm](https://github.com/biowasm/biowasm)** ‚Äì ‚≠ê 255 ‚Äì WebAssembly modules for genomics
- **[ugeneunipro/ugene](https://github.com/ugeneunipro/ugene)** ‚Äì ‚≠ê 253 ‚Äì UGENE is free open-source cross-platform bioinformatics software
