## Trending Bioinformatics Repositories (Updated: 2025-08-29)

**Showing 56 projects matching the criteria (250+ stars, recent updates)**

- **[deepchem/deepchem](https://github.com/deepchem/deepchem)** ‚Äì ‚≠ê 6173 ‚Äì Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology
- **[ArcInstitute/evo2](https://github.com/ArcInstitute/evo2)** ‚Äì ‚≠ê 3046 ‚Äì Genome modeling and design across all domains of life
- **[satijalab/seurat](https://github.com/satijalab/seurat)** ‚Äì ‚≠ê 2500 ‚Äì R toolkit for single cell genomics
- **[materialsproject/pymatgen](https://github.com/materialsproject/pymatgen)** ‚Äì ‚≠ê 1706 ‚Äì Python Materials Genomics (pymatgen) is a robust materials analysis code that defines classes for structures and molecules with support for many electronic structure codes. It powers the Materials Project.
- **[duerrsimon/bioicons](https://github.com/duerrsimon/bioicons)** ‚Äì ‚≠ê 1519 ‚Äì A library of free open source icons for science illustrations in biology and chemistry
- **[MultiQC/MultiQC](https://github.com/MultiQC/MultiQC)** ‚Äì ‚≠ê 1350 ‚Äì Aggregate results from bioinformatics analyses across many samples into a single report.
- **[crazyhottommy/getting-started-with-genomics-tools-and-resources](https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources)** ‚Äì ‚≠ê 1312 ‚Äì Unix, R and python tools for genomics and data science
- **[vgteam/vg](https://github.com/vgteam/vg)** ‚Äì ‚≠ê 1229 ‚Äì tools for working with genome variation graphs
- **[thieu1995/mealpy](https://github.com/thieu1995/mealpy)** ‚Äì ‚≠ê 1076 ‚Äì A Collection Of The State-of-the-art Metaheuristic Algorithms In Python (Metaheuristic/Optimizer/Nature-inspired/Biology)
- **[sib-swiss/training-collection](https://github.com/sib-swiss/training-collection)** ‚Äì ‚≠ê 1054 ‚Äì Collection of bioinformatics training materials
- **[pachterlab/gget](https://github.com/pachterlab/gget)** ‚Äì ‚≠ê 1049 ‚Äì üß¨ gget enables efficient querying of genomic reference databases
- **[hail-is/hail](https://github.com/hail-is/hail)** ‚Äì ‚≠ê 1026 ‚Äì Cloud-native genomic dataframes and batch computing
- **[cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization)** ‚Äì ‚≠ê 1004 ‚Äì A list of interesting genome browser and genome visualization programs
- **[scikit-bio/scikit-bio](https://github.com/scikit-bio/scikit-bio)** ‚Äì ‚≠ê 964 ‚Äì scikit-bio: a community-driven Python library for bioinformatics, providing versatile data structures, algorithms and educational resources.
- **[ablab/spades](https://github.com/ablab/spades)** ‚Äì ‚≠ê 862 ‚Äì SPAdes Genome Assembler
- **[biotite-dev/biotite](https://github.com/biotite-dev/biotite)** ‚Äì ‚≠ê 851 ‚Äì A comprehensive library for computational molecular biology
- **[cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal)** ‚Äì ‚≠ê 848 ‚Äì cBioPortal for Cancer Genomics
- **[tanghaibao/jcvi](https://github.com/tanghaibao/jcvi)** ‚Äì ‚≠ê 846 ‚Äì Python library to facilitate genome assembly, annotation, and comparative genomics
- **[BioContainers/containers](https://github.com/BioContainers/containers)** ‚Äì ‚≠ê 744 ‚Äì Bioinformatics containers
- **[shenwei356/awesome](https://github.com/shenwei356/awesome)** ‚Äì ‚≠ê 701 ‚Äì Awesome resources on Bioinformatics, data science, machine learning, programming language (Python, Golang, R, Perl) and miscellaneous stuff.
- **[igvteam/igv](https://github.com/igvteam/igv)** ‚Äì ‚≠ê 696 ‚Äì Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations
- **[marbl/canu](https://github.com/marbl/canu)** ‚Äì ‚≠ê 690 ‚Äì A single molecule sequence assembler for genomes large and small.
- **[igvteam/igv.js](https://github.com/igvteam/igv.js)** ‚Äì ‚≠ê 684 ‚Äì Embeddable genomic visualization component based on the Integrative Genomics Viewer
- **[zaeleus/noodles](https://github.com/zaeleus/noodles)** ‚Äì ‚≠ê 609 ‚Äì Bioinformatics I/O libraries in Rust
- **[ComparativeGenomicsToolkit/cactus](https://github.com/ComparativeGenomicsToolkit/cactus)** ‚Äì ‚≠ê 598 ‚Äì Official home of genome aligner based upon notion of Cactus graphs
- **[OpenChemistry/avogadrolibs](https://github.com/OpenChemistry/avogadrolibs)** ‚Äì ‚≠ê 551 ‚Äì Avogadro libraries provide 3D rendering, visualization, analysis and data processing useful in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.
- **[Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk)** ‚Äì ‚≠ê 541 ‚Äì GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- **[sourmash-bio/sourmash](https://github.com/sourmash-bio/sourmash)** ‚Äì ‚≠ê 519 ‚Äì Quickly search, compare, and analyze genomic and metagenomic data sets.
- **[lindenb/jvarkit](https://github.com/lindenb/jvarkit)** ‚Äì ‚≠ê 512 ‚Äì Java utilities for Bioinformatics
- **[Ensembl/ensembl-vep](https://github.com/Ensembl/ensembl-vep)** ‚Äì ‚≠ê 501 ‚Äì The Ensembl Variant Effect Predictor predicts the functional effects of genomic variants
- **[SciML/Catalyst.jl](https://github.com/SciML/Catalyst.jl)** ‚Äì ‚≠ê 493 ‚Äì Chemical reaction network and systems biology interface for scientific machine learning (SciML). High performance, GPU-parallelized, and O(1) solvers in open source software.
- **[bactopia/bactopia](https://github.com/bactopia/bactopia)** ‚Äì ‚≠ê 468 ‚Äì A flexible pipeline for complete analysis of bacterial genomes
- **[chrchang/plink-ng](https://github.com/chrchang/plink-ng)** ‚Äì ‚≠ê 463 ‚Äì PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.  This is a comprehensive update to it.
- **[MAGICS-LAB/DNABERT_2](https://github.com/MAGICS-LAB/DNABERT_2)** ‚Äì ‚≠ê 414 ‚Äì [ICLR 2024] DNABERT-2: Efficient Foundation Model and Benchmark for Multi-Species Genome
- **[openvax/pyensembl](https://github.com/openvax/pyensembl)** ‚Äì ‚≠ê 398 ‚Äì Python interface to access reference genome features (such as genes, transcripts, and exons) from Ensembl
- **[zeqianli/tgv](https://github.com/zeqianli/tgv)** ‚Äì ‚≠ê 396 ‚Äì Explore genomes in the terminal. Light, blazing fast üöÄ, vim-motion.
- **[openproblems-bio/openproblems](https://github.com/openproblems-bio/openproblems)** ‚Äì ‚≠ê 386 ‚Äì Formalizing and benchmarking open problems in single-cell genomics
- **[kcleal/gw](https://github.com/kcleal/gw)** ‚Äì ‚≠ê 369 ‚Äì Genome browser and variant annotation
- **[evanpeikon/Bioinformatics_Toolkit](https://github.com/evanpeikon/Bioinformatics_Toolkit)** ‚Äì ‚≠ê 368 ‚Äì A growing collection of bioinformatics tutorials, project guides, tools, and articles I‚Äôve developed to make bioinformatics more accessible.
- **[moshi4/pyGenomeViz](https://github.com/moshi4/pyGenomeViz)** ‚Äì ‚≠ê 356 ‚Äì A genome visualization python package for comparative genomics
- **[ncbi/pgap](https://github.com/ncbi/pgap)** ‚Äì ‚≠ê 353 ‚Äì NCBI Prokaryotic Genome Annotation Pipeline
- **[ebi-pf-team/interproscan](https://github.com/ebi-pf-team/interproscan)** ‚Äì ‚≠ê 343 ‚Äì Genome-scale protein function classification
- **[NVIDIA-Genomics-Research/rapids-single-cell-examples](https://github.com/NVIDIA-Genomics-Research/rapids-single-cell-examples)** ‚Äì ‚≠ê 342 ‚Äì Examples of single-cell genomic analysis accelerated with RAPIDS
- **[fulcrumgenomics/fgbio](https://github.com/fulcrumgenomics/fgbio)** ‚Äì ‚≠ê 341 ‚Äì Tools for working with genomic and high throughput sequencing data.
- **[higlass/higlass](https://github.com/higlass/higlass)** ‚Äì ‚≠ê 332 ‚Äì Fast, flexible and extensible genome browser.
- **[brewsci/homebrew-bio](https://github.com/brewsci/homebrew-bio)** ‚Äì ‚≠ê 318 ‚Äì :beer::microscope: Bioinformatics formulae for the Homebrew package manager (macOS and Linux)
- **[Gaius-Augustus/Augustus](https://github.com/Gaius-Augustus/Augustus)** ‚Äì ‚≠ê 315 ‚Äì Genome annotation with AUGUSTUS
- **[pinellolab/CRISPResso2](https://github.com/pinellolab/CRISPResso2)** ‚Äì ‚≠ê 314 ‚Äì Analysis of deep sequencing data for rapid and intuitive interpretation of genome editing experiments
- **[WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM)** ‚Äì ‚≠ê 293 ‚Äì Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes
- **[wurmlab/sequenceserver](https://github.com/wurmlab/sequenceserver)** ‚Äì ‚≠ê 287 ‚Äì Intuitive graphical web interface for running BLAST bioinformatics tool (i.e. have your own custom NCBI BLAST site!)
- **[projectglow/glow](https://github.com/projectglow/glow)** ‚Äì ‚≠ê 287 ‚Äì An open-source toolkit for large-scale genomic analysis
- **[songlab-cal/gpn](https://github.com/songlab-cal/gpn)** ‚Äì ‚≠ê 277 ‚Äì Genomic Pre-trained Network
- **[labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN)** ‚Äì ‚≠ê 276 ‚Äì Build a partitioned pangenome graph from microbial genomes
- **[mothur/mothur](https://github.com/mothur/mothur)** ‚Äì ‚≠ê 273 ‚Äì Welcome to the mothur project, initiated by Dr. Patrick Schloss and his software development team in the Department of Microbiology & Immunology at The University of Michigan. This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.
- **[biowasm/biowasm](https://github.com/biowasm/biowasm)** ‚Äì ‚≠ê 253 ‚Äì WebAssembly modules for genomics
- **[GMOD/jbrowse-components](https://github.com/GMOD/jbrowse-components)** ‚Äì ‚≠ê 252 ‚Äì Source code for JBrowse 2, a modern React-based genome browser
